crates/ruff/src/rules/isort/categorize.rs

```diff
@@ -43,16 +45,19 @@ pub fn categorize(
     package: Option<&Path>,
     known_first_party: &BTreeSet<String>,
     known_third_party: &BTreeSet<String>,
+    known_local_folder: &BTreeSet<String>,
     extra_standard_library: &BTreeSet<String>,
     target_version: PythonVersion,
 ) -> ImportType {
     let (import_type, reason) = {
         if level.map_or(false, |level| *level > 0) {
-            (ImportType::LocalFolder, Reason::NonZeroLevel)
+            (ImportType::Relative, Reason::NonZeroLevel)
         } else if known_first_party.contains(module_base) {
             (ImportType::FirstParty, Reason::KnownFirstParty)
         } else if known_third_party.contains(module_base) {
             (ImportType::ThirdParty, Reason::KnownThirdParty)
+        } else if known_local_folder.contains(module_base) {
+            (ImportType::LocalFolder, Reason::KnownLocalFolder)
```

Comment:
Doesn't this cause us to render the `.` imports and the "known local folder" imports as separate sections? Does that match isort's behavior? If so I find it a little surprising.

Reply:
Like, I would've expected that we remove `ImportType::Relative`, and just use `ImportType::LocalFolder` in both places.

Reply:
yes, done.

Reply:
Thx